Desktop/property-pro-app/src/pages/Rent/Rent.js

```diff
@@ -0,0 +1,32 @@
+import Footer from "../../components/Footer/Footer";
+import Navbar from "../../components/navbar/Navbar";
+import Property from "../Buy/Property";
+import propertyData from "../Buy/Propertydata";
+import "./rent.css";
+
+
+
+
+
+
+
+const Rent = () => {
+  return <div>
+      <Navbar/>
+
+<h1>Property For Rent</h1>
+      <div className="property_wrapper">
+      {propertyData.map((property) => 
```

Comment:
I think this map item is supposed to have a unique key, try to  work with the browser console for error